### PR TITLE
SW360 VCS curations

### PR DESCRIPTION
**Pull request by MarcelBochtler**

### CODE REVIEW

File: curations/Maven/org.eclipse.sw360/client.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- id: "Maven:org.eclipse.sw360:client"
```
Comment (sschuberth) [inline curations/Maven/org.eclipse.sw360/client.yml:1]: Commit message nit: s/sw360/SW360/

File: curations/Maven/org.eclipse.sw360/client.yml [hunk 2]
```diff
@@ -6,3 +6,10 @@
       type: "Git"
       url: "https://github.com/eclipse/sw360.git"
       path: "clients/client"
+
```
Comment (sschuberth) [inline curations/Maven/org.eclipse.sw360/client.yml:9]: Commit message nit: s/sw360/SW360/

Also, I believe the order of words is somehow off... maybe better

> curations: Add VCS revisions for some SW360 http-support and client versions

File: curations/Maven/org.eclipse.sw360/client.yml [hunk 3]
```diff
@@ -6,3 +6,10 @@
...
+- id: "Maven:org.eclipse.sw360:client:13.2.0-28"
+  curations:
+    comment: |
+      Unfortunately, the version can not automatically determined by ORT.
+    vcs:
+      revision: "sw360-13.2.0-M1"
```
Comment (sschuberth) [inline curations/Maven/org.eclipse.sw360/client.yml:15]: Hmm, don't the `-28` versions denote weekly builds (for week 28 in this case), which is different from `M` == "milestone" builds?

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: curations/Maven/org.eclipse.sw360/client.yml (hunk 3 above)
Comment (MarcelBochtler) [reply curations/Maven/org.eclipse.sw360/client.yml:15]: That is very unfortunate :(
I'll drop that commit.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**